admin/class-acc_user_importer-admin.php

```diff
@@ -612,7 +640,7 @@ public function log_local_output( $v ) {
 			//If it's a new run of the script, evaluate which log file to use
 			//and cache it for next time around for efficiency.
 			if ($new_run) {
-				$log_directory  = WP_CONTENT_DIR . '/acc/logs/';
+				$log_directory  = plugin_dir_path . 'logs/acc/';
```

Comment:
Hmm, I don't think this works? I assume you wanted to use the [plugin_dir_path](https://developer.wordpress.org/reference/functions/plugin_dir_path/) function, but that requires an argument e.g. `plugin_dir_path(__FILE__)` would work. That said, this would tie the log location to the location of this file, thus moving the logs to a new directory if someone would refactor this file. 
I purposely picked `WP_CONTENT_DIR` as a root for the logs to detach it from the code structure, why did you change it? Was it broken?